x-pack/plugins/alerting/server/rules_client/rules_client.ts

```diff
@@ -2652,6 +2652,52 @@ export class RulesClient {
     }
   }
 
+  public async runSoon({ id }: { id: string }) {
+    const { attributes } = await this.unsecuredSavedObjectsClient.get<Rule>('alert', id);
+    const taskDoc = await this.taskManager.get(id);
```

Comment:
Prior to 8.0, the task document had a random ID that was generated and then stored on the rule under `scheduledTaskId`. I think there's a small chance this could still be the case, so I would use the `scheduledTaskId` from the rule SO attributes for the task manager `get` instead of the ID

Reply:
Resolved in https://github.com/elastic/kibana/pull/139848/commits/8d739ac9c783f035e6dba8bd3dfe1b20425b173f